roles/jd-client/README.md

```diff
@@ -9,30 +11,28 @@ After the setup phase it will start to negotiate jobs with upstream and send the
 downstream, so that everything that is dowstream do not need to know that job is negotiated.
 
 ## Setup
+
 ### Configuration File
-The `proxy-config-example.toml` is a configuration example.
 
 The configuration file contains the following information:
 
-1. The Upstream connection information which includes the SV2 Pool authority public key 
-   (`upstream_authority_pubkey`) and the SV2 Pool connection address (`upstream_address`) and port
-   (`upstream_port`).
-2. The maximum and minimum SV2 versions (`max_supported_version` and `min_supported_version`)
-3. The Job Declarator information which includes the Pool JD connection address (`jd_address`) and the Template Provider connection address to which to connect (`tp_address`).
-4. Optionally, you may want to verify that your TP connection is authentic. You may get `tp_authority_public_key` from the logs of your TP, for example:
-```
-# 2024-02-13T14:59:24Z Template Provider authority key: EguTM8URcZDQVeEBsM4B5vg9weqEUnufA8pm85fG4bZd
-```
+1. The downstream connection information which includes  connection address (`downstream_address`) and port (`downstream_port`).
+2. The maximum and minimum SRI versions (`max_supported_version` and `min_supported_version`) with size as (`min_extranonce2_size`)
+3. In this file, there is a withhold (`withhold`) with a booloan value.
+4. The authentication keys for open encrypted connection for the downstream (`authority_public_key`, `authority_secret_key` and `cert_validity_sec`)
+5. The retry that tells JDC the number of times to retry itself after a failure.
+6. The Job Declarator information which includes the Template Provider connection address to which to connect (`tp_address`).
+7. Optionally, you may want to verify that your TP connection is authentic. You may get `tp_authority_public_key` from the logs of your.
```

Comment:
```suggestion
1. The downstream connection information which includes connection address (`downstream_address`) and port (`downstream_port`).
2. The maximum and minimum SRI versions (`max_supported_version` and `min_supported_version`).
3. Minimum `extranonce2` size for downstream (`min_extranonce2_size`).
4. A `withhold` boolean, which prevents a valid block from being submitted upstream.
5. The authentication keys for the encrypted connection downstream (`authority_public_key`, `authority_secret_key` and `cert_validity_sec`)
6. The `retry`, which tells JDC the number of times to reinitialize itself after a failure.
7. The Template Provider connection address to which to connect (`tp_address`). Optionally, you may also want to verify that your TP connection is authentic. You may get `tp_authority_public_key` from the logs of your TP `bitcoind`.
8. A List of coinbase outputs (`coinbase_outputs`) used to build the coinbase tx in case of Solo Mining (as a fallback system).
9. The `timeout` limit to try to reconnect to upstream.
10. Upstream connection info (`authority_pubkey`, `pool_address`, `jd_address`, `pool_signature`)
```